3_javascript/javascript1/week3/JS1-W3

```diff
@@ -0,0 +1,121 @@
+////////////////////////////Item array removal
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
+  }
+}
+
+console.log(names);
+
+////////////////////////////When will we be there??
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+  function timeCalculator() {
+    let travelTimeInHours = travelInformation.destinationDistance / travelInformation.speed;
+    const hours = Math.floor(travelTimeInHours);
+    const minutes = Math.round((travelTimeInHours - hours) * 60);
+    
+    return {
+      hours,
+      minutes,
+    };
+  }
+  
+  const { hours, minutes } = timeCalculator(travelInformation);
+  
+  console.log(`${hours} hours and ${minutes} minutes`);
+  
+////////////////////////////Series duration of my life
+
+const seriesDurations = [
+    {
+      title: "Game of thrones",
+      days: 3,
+      hours: 1,
+      minutes: 0,
+    },
+    {
+      title: "Handmaids Tale",
+      days: 1,
+      hours: 22,
+      minutes: 40,
+    },
+    {
+      title: "Westworld",
+      days: 3,
+      hours: 10,
+      minutes: 40,
+    },
+  ];
+
+  function logOutSeriesText() {
+    let totalMinutesWatched = 0;
+
+    for (const series of seriesDurations) {
+      const { days, hours, minutes } = series;
+      const seriesDurationInMinutes = days * 24 * 60 + hours * 60 + minutes;
+      totalMinutesWatched += seriesDurationInMinutes;
+
+      const yearsWatched = totalMinutesWatched / (60 * 24 * 365); 
+      const percentageOfLife = (yearsWatched / 80) * 100; 
+
+      console.log(
+        `${series.title} took ${percentageOfLife.toFixed(2)}% of my life.`
+      );
+    }
+
+    console.log(`In total, that is ${totalMinutesWatched} minutes of my life.`);
+  }
+logOutSeriesText();
+
+////////////////////////////Note app
+
+const notes = [];
+
+function saveNote(content, id) {
+    notes.push({ content: content, id: id });
+  }
+
+  saveNote("Pick up groceries", 1);
+  saveNote("Do laundry", 2);
+
+  console.log(notes);
+
+  /////get the note
+
+  function getNote(id) {
+    for (i = 0; i < notes.length; i++){
+        if(notes[i].id = id){
+            return notes[i]
+        } return "Error!"
+    } 
+}
+  
+  const firstNote = getNote(1);
+  console.log(firstNote); 
+
+  /////log out 
+
+  function logOutNotesFormatted() {
+    for(i=0; i< notes.length-1; i++){
+    console.log(`The note with id: ${notes[i].id}, has the following note text: ${notes[i].content}`);
+    }
+  }
+   logOutNotesFormatted(); 
```

Comment:
🌠 The `saveNote()` function is simple and effective! Brilliantly done!
🔍 There's a tiny hitch in the `getNote()` function. Instead of a single equals sign `(=)`, which assigns the value, you should use the comparison `(== or ===)`.
🤖 For your `logOutNotesFormatted()` function, there's a small oversight. Using `i < notes.length-1` might exclude the last note. Swapping it to `i < notes.length` should fix it.
🚀 Consistency Alert! Remember to use `let` or `const` when initializing the loop variable `i`.